app/actions/DexActions.js

```diff
@@ -462,6 +465,14 @@ export const launchDexWindow = () => async (dispatch, getState) => {
     const serverAddress = dexServerAddress;
     await dex.launchWindow(serverAddress);
     dispatch({ type: DEX_LAUNCH_WINDOW_SUCCESS });
+    if (!dexReady) {
+      const walletConfig = wallet.getWalletCfg(
+        sel.isTestNet(getState()),
+        walletName
+      );
+      walletConfig.set(configConstants.DEX_READY, true);
```

Comment:
Maybe I misunderstood something, but I thought this should be called from the `loginDex` function to show login once at least

Reply:
nm. I get it now